tests/test_api/test_submit_job.py

```diff
@@ -266,6 +266,8 @@ def test_submit_good_autorift_granule_names(client, tables):
         'LO08_L1GT_043001_20201106_20201110_02_T2',
         'LC09_L1GT_215109_20220125_20220125_02_T2',
         'LO09_L1GT_215109_20220210_20220210_02_T2',
+        'S2B_22XEQ_20190610_11_L1C',
+        'S2A_22XEQ_20190610_10_L1C',
```

Comment:
Consider moving these up to line 265 with the other S2 examples, which would nicely visualize that we're trying each of 23, 24, and 25 characters?